src/components/ListItem.tsx

```diff
@@ -1,31 +1,27 @@
-import { getDownloadURL } from "firebase/storage";
-import { useEffect, useState } from "react";
-import { getPathReference } from "../firebase/config";
-import { LocationType } from "../types";
+import { PointType } from "../types";
+import { ButtonLink } from "./ButtonLink";
+import { ROUTES } from "../routes";
 
 interface Props {
-  location: LocationType;
+  point: PointType;
 }
 
-export const ListItem = ({ location }: Props) => {
-  const [imageUrl, setImageUrl] = useState<string>("");
-  useEffect(() => {
-    const getImageUrl = async () => {
-      const url = await getDownloadURL(getPathReference(location.imagesrc));
-      setImageUrl(url);
-    };
-    getImageUrl();
-  }, [location.imagesrc]);
+export const ListItem = ({ point }: Props) => {
```

Comment:
Tady to prosim jen nekam poznamenej, ye je potreba to potom doupravit at to vypada, tak jak ma :)